action.yml

```diff
@@ -10,6 +10,14 @@ inputs:
   keep_latest:
     description: how many latest releases to keep
     required: true
+  keep_min_download_counts:
+    description: keep old releases with downloads greater than this parameter <
+    required: false
+    default: 0
+  delete_expired_data:
+    description: Delete data older than a specified number of days
```

Comment:
Hmm "data" is rather ambiguous?  Perhaps we describe with "...tags, releases, drafts and each ones attached assets"?